src/app/application.cpp

```diff
@@ -278,7 +279,7 @@ void Application::torrentFinished(BitTorrent::TorrentHandle *const torrent)
         program.replace("%T", torrent->currentTracker());
         program.replace("%I", torrent->hash());
 
-        QProcess::startDetached(program);
+        std::thread ([program]() { return system(program.toStdString().c_str()); }).detach();
```

Comment:
>Remove space between std::thread and (.

OK.

>We can't exit qBittorrent when any thread is running.

Can you explain more about it?
In linux, I tried the script `sleep 10000s` then exit qbt. The `sleep` process still kept running.